src/Controls/tests/UITests/Tests/Issues/Issue18740.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18740 : _IssuesUITest
+	{
+		public Issue18740(TestDevice device)
+			: base(device)
+		{ }
+
+		public override string Issue => "Virtual keyboard appears with focus on Entry";
+
+		[Test]
+		public void Issue18740Test()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+			App.WaitForElement("WaitForStubControl");
+
+			// 1.Make sure keyboard starts out closed.
+			Assert.IsFalse(App.IsKeyboardShown());
```

Comment:
If the keyboard here is shown, can we just close it instead of doing an assert? All we care about really is that the keyboard is closed before the "enterText" line runs.